### DISCUSSION TITLE
Cannot open UTM-HV by Trollstore2 / 16.1 / M1

### DISCUSSION THREAD

**Discussion by dev-wantap** [utmapp/UTM#5959] [UTM Help (iOS)] [answered]
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

Cannot open UTM(UTM-HV). 
When I try to run the app, it just shuts down without any message.

**Configuration (required)**
* UTM Version: UTM-HV 4.4.5
* OS Version: iPadOS 16.1 (20B82)
* Device Model: A2378 (iPad Pro 12.9 5th gen / M1)
* Is it jailbroken (name jailbreak used)? No
* How did you install UTM? by Trollstore2

**Crash log**
`Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.
Cannot find crash log

**Comment by osy** [comment]
Please provide a crash log

**Comment by dev-wantap** [comment]
> Please provide a crash log

I tried to find a crash log at Privacy -> Analytics & Improvements -> Analytics Data, but there's no log for UTM

**Comment by osy** [comment]
Any chance you can connect it to a computer with Xcode and look in Console and filter by UTM when trying to launch it?

**Comment by dev-wantap** [comment]
> Any chance you can connect it to a computer with Xcode and look in Console and filter by UTM when trying to launch it?

I don't have a mac now, so I think it will be difficult. 
I'll try it if I can get a mac os device.

**Comment by osy** [comment]
Curious if the unofficial build works https://github.com/xiaoxi654/github-action/releases/download/UTM/UTM-HV-iOS16.ipa

**Comment by dev-wantap** [comment]
> Curious if the unofficial build works https://github.com/xiaoxi654/github-action/releases/download/UTM/UTM-HV-iOS16.ipa

It doesn't work (same as HV 4.4.5 version)

And I tried 4.4.4, 4.4.3, but they all don't work

**Comment by osy** [comment]
I think this is a TrollStore issue

**Reply by dev-wantap** [reply]
Then I'll wait for the Trollstore update. Thanks 🙂

**Reply by osy** [reply]
~~https://github.com/opa334/TrollStore/commit/bbdd0fdcdd47199c14614bb76cfe5e7eec251283 should fix it when it is released I think~~

Nvm, I think it's unreleased

**Reply by dev-wantap** [reply, marked as answer]
Updated the trollstore to 2.0.11
UTM-HV runs well.
(This version can activate developer mode on its own)
